ui/keys/prKeys.go

```diff
@@ -74,3 +75,45 @@ func PRFullHelp() []key.Binding {
 		PRKeys.WatchChecks,
 	}
 }
+
+func rebindPRKeys(keys []config.Keybinding) error {
```

Comment:
I see there's the `SetKeys` method on the KeyBinding so maybe you can update the global `Keys` struct that way?